dll/shellext/fontext/CFontMenu.cpp

```diff
@@ -69,20 +69,9 @@ static void RunFontViewer(HWND hwnd, const FontPidlEntry* fontEntry)
     WCHAR FontViewerPath[MAX_PATH] = L"%SystemRoot%\\System32\\fontview.exe";
```

Comment:
Shouldn't it use whatever is specified in the registry to open the font files?

Reply:
Completely unrelated to this PR